web/modules/mof/mof-licenses.json

```
@@ -355,6 +355,33 @@
          "isFsfLibre":true,
          "ContentType":"document"
       },
+      {
+         "reference":"https://spdx.org/licenses/CC-BY-SA-3.0.html",
+         "isDeprecatedLicenseId":false,
+         "detailsUrl":"https://spdx.org/licenses/CC-BY-SA-3.0.json",
+         "referenceNumber":621,
+         "name":"Creative Commons Attribution Share Alike 3.0 Unported",
+         "licenseId":"CC-BY-SA-3.0",
+         "seeAlso": [
+           "https://creativecommons.org/licenses/by-sa/3.0/legalcode"
+         ],
+         "isOsiApproved": false,
+         "ContentType":"document"
+      },
+      {
+         "reference":"https://spdx.org/licenses/CC-BY-SA-4.0.html",
+         "isDeprecatedLicenseId":false,
+         "detailsUrl":"https://spdx.org/licenses/CC-BY-SA-4.0.json",
+         "referenceNumber":155,
+         "name":"Creative Commons Attribution 4.0 International",
```

The name isn't right. This should be instead:
```suggestion
         "name":"Creative Commons Attribution Share Alike 4.0 International",
```